Runtime/AvatarCreator/WebRequests/AvatarAPIRequests.cs

```diff
@@ -209,10 +222,15 @@ public async Task<string> SaveAvatar(string avatarId)
 
         public async Task DeleteAvatar(string avatarId, bool isDraft = false)
```

Comment:
Yup, I thought the same thing about adding another method (since they are technically different endpoints) but I didn't want to change any specific logic in this PR outside of removing the endpoint files, since that is what I wanted to demonstrate.